ext/ffi/ffi.c

```diff
@@ -898,7 +898,11 @@ static void *zend_ffi_create_callback(zend_ffi_type *type, zval *value) /* {{{ *
 	}
 
 	if (!zend_is_callable_ex(value, NULL, 0, NULL, &fcc, &error)) {
-		zend_throw_error(zend_ffi_exception_ce, "Attempt to assign an invalid callback, %s", error);
+		if(n >= 1000) {
```

Comment:
1000 is pretty arbitrary here. If you need to use a sentinel value, please use `(uint32_t) -1`.